### DISCUSSION TITLE
Sizing a DB for Grafana

### DISCUSSION THREAD

**Discussion by Kav957** [grafana/grafana#57932] [General]
How do you estimate the size of a database required before deploying Grafana?

I'm designing a large scale Grafana cluster using Postgres as the database. The compute layer requirements is well documented. However, I cannot find any documentation on sizing a database.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
